zip-0320.rst

```diff
@@ -203,13 +202,20 @@ will be replaced by:
   such that a Unified Address MUST **either**:
 
   * contain at least one shielded Receiver (Typecodes :math:`\mathtt{0x02}`
-    and :math:`\mathtt{0x03}`), and no Traceable P2PKH Receiver (Typecode
-    :math:`\mathtt{0x04}`); **or**
-  * contain **only** a Traceable P2PKH Receiver (Typecode :math:`\mathtt{0x04}`).
+    and :math:`\mathtt{0x03}`), and **no** Source Restriction Metadata Item
+    having value :math:`\mathtt{0x00}`; **or**
+  * contain **both** a Transparent Receiver (Typecode :math:`\mathtt{0x00}` or
+    Typecode :math:`\mathtt{0x01}`) **and** a Source Restriction Metadata Item
+    (Typecode :math:`\mathtt{0xE2}`) having value :math:`\mathtt{0x00}`, and
+    **no other** Receivers.
 
   A Unified Viewing Key MUST contain at least one shielded Item (Typecodes
   :math:`\mathtt{0x02}` and :math:`\mathtt{0x03}`). 
```

Comment:
```suggestion
  :math:`\mathtt{0x02}` and :math:`\mathtt{0x03}`), and **no** Source
  Restriction Metadata Item.
```
This excludes even future Source Restriction Metadata Item values from being applicable to UVKs. I think that's what we want.